### PR TITLE
Upgrade rails 6 to latest

### LINKED ISSUE #3810: upgrade to latest rails 6.1 version
We should ride the latest version of 6

### PR DESCRIPTION
**Pull request by mattlindsey**
fixes #3810

### CODE REVIEW

**Comment by mattlindsey** [comment]
Changed 'participants' from text to JSON, but still getting an error here:

rspec ./spec/presenters/event_instance_presenter_spec.rb

File: Dockerfile [hunk 1]
```diff
@@ -20,22 +20,20 @@ ENTRYPOINT ["/docker-entrypoint.sh"]
 #set it "correctly" for now. perhaps replace phantomjs with something else?
 ENV BUNDLE_PATH=/bundle \
     BUNDLE_BIN=/bundle/bin \
-    GEM_HOME=/bundle \
-    OPENSSL_CONF=/dev/null
+    GEM_HOME=/bundle
+#    OPENSSL_CONF=/dev/null
```
Comment (mtc2013) [inline Dockerfile:24]: Is the docker npx jasmine-browser-runner still working with this change?

Comment (mtc2013) [reply Dockerfile:24]: Checked this just now and it works.

**Review by mtc2013 — APPROVED** [review]
We're good.

**Pull request merged**